website/content/docs/commands/plugin/register.mdx

```diff
@@ -11,9 +11,17 @@ description: |-
 The `plugin register` command registers a new plugin in Vault's plugin catalog.
 The plugin's type of "auth", "database", or "secret" must be included.
 
+<Note>
+
+To register an Enterprise plugin, you must register with the plugin artifact, not the binary.
```

Comment:
```suggestion
To register an enterprise plugin, you must register with the extracted
`.zip` file, not the binary.
```
Technically "artifact" could refer to anything. Even a binary file can be an artifact in some cases. Since we mean "extracted .zip file", it's clearer for readers if we actually say that

Reply:
It's up to you if you want to apply the full suggestion or just the capitalization change. I'm realizing I probably missed the "artifact" usage in the original PR, so feel free to ignore that part so the docs stay consistent. We/I can go back to tweak the language later